README.md

```diff
@@ -4,6 +4,10 @@
 With the help of
 
 ![Powered by PhpStorm](https://www.jetbrains.com/phpstorm/documentation/docs/logo_phpstorm.png)
+
+##Requirements##
+To load this library, you must have [Composer](https://getcomposer.org) installed and operational. The Spoofchecking validation requires that your PHP system have the [PHP Internationalization Libraries](http://php.net/manual/en/book.intl.php) (also known as PHP Intl) insalled.
```

Comment:
Can you please make it a list?
Also add a link to the Spoof validation or to the list of available validations.